### DISCUSSION TITLE
old custom suricata not showing at detection page

### DISCUSSION THREAD

**Discussion by majungman** [Security-Onion-Solutions/securityonion#13307] [2.4] [answered]
### Version

2.4.80

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

100

### RAM

100

### Storage for /

1T

### Storage for /nsm

1T

### Network Traffic Collection

span port

### Network Traffic Speeds

more than 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

hi previously i have added new custom rules at the configuration page now it is not able to edit ( i want to delete some testing rules)  <img width="477" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/102951771/d87f75d2-8bab-4e6f-bd68-eb1feeb8869b"> and this is not showing at the detection page
<img width="391" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/102951771/bbdedc72-d7eb-4158-8b5f-6748151c7fc2">

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by defensivedepth** [comment, marked as answer]
You will need to remove those rules from /opt/so/saltstack/local/salt/idstools/rules/local.rules and then re-add them using the Detections interface. You can do that individually or bulk.

Also see here for more details: https://github.com/Security-Onion-Solutions/securityonion/discussions/13206#discussioncomment-9970194
